elfi/bo/acquisition.py

```diff
@@ -42,15 +42,15 @@ def evaluate(self, x, t=None):
         """
         return NotImplementedError
 
-    def acquire(self, n_values, pending_locations=None, t=None):
+    def acquire(self, n_values, pending_locations=None, t=0):
```

Comment:
Yes, check is fine. I guess it was None at first because not all methods use it. The check can be implemented in the evaluate function though.